### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by LDigital84** [zurb/foundation-sites-6#28]
I am attempting to get started with FS6 but having an issue. I am sure it is me or my computer but I need some help. I am using Windows 8.1 using Git Bash (running as Administrator). NodeJS is v4.1.

I don't do this that often so I am unsure how to solve my issue. I start by doing the following: 

``` bash
git clone https://github.com/zurb/foundation-sites-6.git
cd foundation-sites-6
```

Works fine. I then do `npm install && bower install` and I get the long list of various warnings and errors. I would ignore them but when I try running `npm start`  it tells me `Error: Cannot find module 'browser-sync'`

How do I solve this problem? 

I appreciate the help, while I am good with HTML/CSS/JS, the rest of this isn't my area so I am a bit lost. 

What I see when I run `npm install && bower install`

```
npm WARN peerDependencies The peer dependency grunt@^0.4.5 included from bootcamp will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN peerDependencies The peer dependency grunt-contrib-sass@^0.8.0 included from bootcamp will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN optional dep failed, continuing fsevents@1.0.0

> utf-8-validate@1.2.1 install C:\Users\Lynda\foundation-sites-6\node_modules\browser-                                                                                                                                        sync\node_modules\socket.io\node_modules\engine.io\node_modules\ws\node_modules\utf-8-validate
> node-gyp rebuild


C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\engine.io\node_modules\ws\node_modules\utf-8-validate>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB3428: Could not load the Visual C++ component "VCBuild.exe". To fix this, 1) install the .NET Framework 2.0 SDK, 2) install Microsoft Visual Studio 2005 or 3) add the location of the component to the system path if it is installed elsewhere.  [C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.i                                                                                                                                        o\node_modules\engine.io\node_modules\ws\node_modules\utf-8-validate\build\binding.sln]
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\no                                                                                                                                        de_modules\node-gyp\lib\build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:2                                                                                                                                        00:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\nod                                                                                                                                        e_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\                                                                                                                                        socket.io\node_modules\engine.io\node_modules\ws\node_modules\utf-8-validate
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok

> bufferutil@1.2.1 install C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync                                                                                                                                        \node_modules\socket.io\node_modules\engine.io\node_modules\ws\node_modules\bufferutil
> node-gyp rebuild


C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\nod                                                                                                                                        e_modules\engine.io\node_modules\ws\node_modules\bufferutil>if not defined npm_config_                                                                                                                                        node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_                                                                                                                                        modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB3428: Could not load the Visual C++ component "VCBuild.exe". To fix this, 1) install the .NET Framework 2.0 SDK, 2) install Microsoft V                                                                    isual Studio 2005 or 3) add the location of the component to the system path if it is installed elsewhere.  [C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\engine.io\node_modules\ws\node_modules\bufferutil\build\binding.sln]
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\engine.io\node_modules\ws\node_modules\bufferutil
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok
npm WARN optional dep failed, continuing utf-8-validate@1.2.1

> utf-8-validate@1.2.1 install C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\utf-8-validate
> node-gyp rebuild


C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\utf-8-validate>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB3428: Could not load the Visual C++ component "VCBuild.exe". To fix this, 1) install the .NET Framework 2.0 SDK, 2) install Microsoft V                                                                    isual Studio 2005 or 3) add the location of the component to the system path if it is installed elsewhere.  [C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\utf-8-validate\build\binding.sln]
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\utf-8-validate
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok
npm WARN optional dep failed, continuing bufferutil@1.2.1

> bufferutil@1.2.1 install C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\bufferutil
> node-gyp rebuild


C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\bufferutil>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node  rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
MSBUILD : error MSB3428: Could not load the Visual C++ component "VCBuild.exe". To fix this, 1) install the .NET Framework 2.0 SDK, 2) install Microsoft V                                                                    isual Studio 2005 or 3) add the location of the component to the system path if it is installed elsewhere.  [C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\bufferutil\build\binding.sl                                                                    n]
gyp ERR! build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Lynda\foundation-sites-6\node_modules\browser-sync\node_modules\socket.io\node_modules\socket.io-client\node_modules\engine.io-client\node_modules\ws\node_modules\bufferutil
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok
npm WARN optional dep failed, continuing utf-8-validate@1.2.1

> spawn-sync@1.0.13 postinstall C:\Users\Lynda\foundation-sites-6\node_modules\gulp-sass\node_modules\node-sass\node_modules\cross-spawn\node_modules\spawn-sync
> node postinstall

npm WARN optional dep failed, continuing bufferutil@1.2.1

> node-sass@3.3.3 install C:\Users\Lynda\foundation-sites-6\node_modules\gulp-sass\node_modules\node-sass
> node scripts/install.js

Binary downloaded and installed at C:\Users\Lynda\foundation-sites-6\node_modules\gulp-sass\node_modules\node-sass\vendor\win32-x64-46\binding.node

> node-sass@3.3.3 postinstall C:\Users\Lynda\foundation-sites-6\node_modules\gulp-sass\node_modules\node-sass
> node scripts/build.js

` C:\Users\Lynda\foundation-sites-6\node_modules\gulp-sass\node_modules\node-sass\vendor\win32-x64-46\binding.node ` exists.
 testing binary.
Binary is fine; exiting.
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.1.0
npm ERR! npm  v2.14.3
npm ERR! path C:\Users\Lynda\AppData\Roaming\npm-cache\xtend\4.0.0\package\package.json.aee12beeae6b6c2eca44e55179d8e462
npm ERR! code EPERM
npm ERR! errno -4048
npm ERR! syscall rename

npm ERR! Error: EPERM: operation not permitted, rename 'C:\Users\Lynda\AppData\Roaming\npm-cache\xtend\4.0.0\package\package.json.aee12beeae6b6c2eca44e55179d8e462' -> 'C:\Users\Lynda\AppData\Roaming\npm-cache\xtend\4.0.0\package\package.json'
npm ERR!     at Error (native)
npm ERR!  { [Error: EPERM: operation not permitted, rename 'C:\Users\Lynda\AppData\Roaming\npm-cache\xtend\4.0.0\package\package.json.aee12beeae6b6c2eca44e55179d8e462' -> 'C:\Users\Lynda\AppData\Roaming\npm-cache\xtend\4.0.0\package\package.json']
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'rename',
npm ERR!   path: 'C:\\Users\\Lynda\\AppData\\Roaming\\npm-cache\\xtend\\4.0.0\\package\\package.json.aee12beeae6b6c2eca44e55179d8e462',
npm ERR!   dest: 'C:\\Users\\Lynda\\AppData\\Roaming\\npm-cache\\xtend\\4.0.0\\package\\package.json',
npm ERR!   parent: 'through2' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Lynda\foundation-sites-6\npm-debug.log
```

What I see after trying to run `npm start`: 

```
> foundation-sites@6.0.0-alpha.1 start C:\Users\Lynda\foundation-sites-6
> gulp

module.js:338
    throw err;
    ^

Error: Cannot find module 'browser-sync'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (C:\Users\Lynda\foundation-sites-6\gulpfile.js:2:15)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v4.1.0
npm ERR! npm  v2.14.3
npm ERR! code ELIFECYCLE
npm ERR! foundation-sites@6.0.0-alpha.1 start: `gulp`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the foundation-sites@6.0.0-alpha.1 start script 'gulp'.
npm ERR! This is most likely a problem with the foundation-sites package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get their info via:
npm ERR!     npm owner ls foundation-sites
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Lynda\foundation-sites-6\npm-debug.log

```

**Comment by 4shoda** [comment]
I'm having a similar issue - gulp errors "cannot find module...". Based on your conversation, I find it odd that I am getting this error because I do have Visual Studio 2010 installed? But I suppose I might have only installed some of Visual Studio. 

So, I'm now installed all the MinGW packages that have GCC in the title. But still I get the error. What am I missing? Do I need to stop / start some process? Reboot didn't seem to do it. Somehow I am missing something in this process. Can anyone help?

**Comment by gakimball** [comment]
@4shoda Are you getting the same error? I believe `Cannot find module x` is going to be a different error than Libsass failing to build.

**Comment by 4shoda** [comment]
Yes and I'm stuck. I posted a question on the Foundation forum but so I haven't found an answer. I started over from scratch on a different computer and same issue - even with the cloned Foundation 6 project. Just hoping it is resolved soon as I would love to start working with this. Maybe as more Windows folks work with Foundation 6 there will be more information on it? That's the hope anyways. 

Let me know if anyone has any ideas. 

Thanks!
